Makefile

```diff
@@ -316,10 +316,10 @@ endif
 
 # Debugging
 ifeq ($(DEBUG), 1)
-	COMMON_FLAGS += -DDEBUG -g -O0
+	COMMON_FLAGS += -DDEBUG -g -O0 -std=c++11
 	NVCCFLAGS += -G
 else
-	COMMON_FLAGS += -DNDEBUG -O2
+	COMMON_FLAGS += -DNDEBUG -O2 -std=c++11
```

Comment:
When I do `man gcc`  on a `g2.2xlarge`, the default is `gnu++98`. It says "GNU dialect of -std=c++98.  This is the default for C++ code."

We can discuss how to check for breaking changes.